fennec_aurora_task_creator/config.py

```diff
@@ -172,3 +169,12 @@ def _set_dict_path(recursive_dictionary, path_list, value):
 
 def _get_dict_path(dictionary, path_list):
     return dictionary[path_list[0]] if len(path_list) == 1 else _get_dict_path(dictionary[path_list[0]], path_list[1:])
+
+
+def _convert_value_to_correct_type(value, default_value=None):
+    type_of_default_value = type(default_value)
```

Comment:
Okay! I don't believe I can use `isinstance()`, because `type_of_default_value` is used to reinstantiate the correct primitive type, a couple of lines below.